app/src/main/java/com/dessalines/thumbkey/keyboards/JAKanaThumbKey.kt

```diff
@@ -105,7 +105,7 @@ val KB_JA_KANA_THUMBKEY_HIRAGANA =
                     center =
                         KeyC(
                             display = KeyDisplay.TextDisplay("ア"),
-                            action = ToggleCapsLock,
+                            action = ShiftAndCapsLock(true),
```

Comment:
If enable is always true, you might as well remove it. Unless you plan on adding another one that disables shift mode.